sdlf-team/scripts/bootstrap_team.sh

```diff
@@ -42,6 +59,7 @@ if ! aws cloudformation describe-stacks --stack-name ${STACK_NAME}; then
         ParameterKey=pChildAccountId,ParameterValue="${CHILD_ACCOUNT}" \
         ParameterKey=pEnvironment,ParameterValue="${ENV}" \
         ParameterKey=pTeamName,ParameterValue="${TEAM_NAME}" \
+        ParameterKey=pSNSNotificationsEmail,ParameterValue=\"${NOTIFICATION_EMAIL}\"
```

Comment:
A "\\" is missing at the end of this line. It should be:
`ParameterKey=pSNSNotificationsEmail,ParameterValue=\"${NOTIFICATION_EMAIL}\"` \